packages/form-fields/select/style.module.css

```diff
@@ -0,0 +1,30 @@
+.root {
+  composes: wpl-variables input-root from '../style.module.css';
+  width: 100%;
+  margin-bottom: 24px;
+
+  &.hidden {
+    display: none;
+  }
+}
+
+.theme-dark {
+  composes: wpl-variables-dark from '../style.module.css';
+}
+
+.input {
+  composes: wpl-input-base from '../style.module.css';
+  appearance: none;
+  background-image: url("data:image/svg+xml,%3csvg xmlns='http://www.w3.org/2000/svg' viewBox='0 0 16 16'%3e%3cpath fill='none' stroke='%23343a40' stroke-linecap='round' stroke-linejoin='round' stroke-width='2' d='m2 5 6 6 6-6'/%3e%3c/svg%3e");
```

Comment:
Are we able to use currentColor as the chevron value? On dark backgrounds the chevron is invisible.

<img width="194" alt="CleanShot 2022-11-22 at 11 30 19@2x" src="https://user-images.githubusercontent.com/825855/203369049-bb03e004-d2b0-4234-9374-bb4c6daa56e4.png">

Reply:
Can't use `currentColor` since the chevron is a data URL, but it's easy enough to add in a `.theme-dark` specific version!